ReplicationAcceptance/ReplicationAcceptance/CloudantReplicationBase.m

```diff
@@ -12,11 +12,15 @@
 #import "CloudantSyncEncryption.h"
 
 #import "ReplicationSettings.h"
+#import "CDTLogging.h"
```

Comment:
Looks like you can back out all of the changes in this file (unneeded import and whitespace)